packages/sanes-chrome-extension/src/routes/restore-account/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import RestoreAccountForm from './components';
+import { FormValues } from 'medulas-react-components/lib/components/forms/Form';
+import { history } from '../../store/reducers';
+import { PersonaManager } from '../../logic/persona';
+import { PersonaContext } from '../../context/PersonaProvider';
+
+const onBack = (): void => {
+  history.goBack();
+};
+
+const RestoreAccount = (): JSX.Element => {
+  const personaProvider = React.useContext(PersonaContext);
+
+  const onRestore = async (formValues: FormValues): Promise<void> => {
+    try {
+      const persona = await PersonaManager.create();
```

Comment:
You should pass the `mnemonic` obtained from form, to `await PersonaManager.create()` otherwise, you will create a different persona.